### ISSUE TITLE
pass GPX files to open at the command line

### ISSUE THREAD

**Issue by IzzySoft** [tboegi/GpsMaster#18]
Is it somehow possible to pass a (GPX) file to be opened right at the command line, e.g.

    java -jar GpsMaster.jar myfile.gpx

I tried that, but it was ignored. I also tried to find out whether there are any options accepted via `--help` to no avail. As I mostly work from terminal, it would be very helpful to open a file I'm just at – or to even make GpsMaster the "default application" for opening `*.gpx` files.

**Comment by rfuegen** [comment]
the functionality to load GPX files via command line is not implemented yet. but since this has been requested by some users, I try to implement it ASAP. please be patient.

**Comment by IzzySoft** [comment]
Oh, I will – thanks a lot! I was in no way trying to "press". I'll patiently wait for your (closure) note here :smiley: Best luck! And thanks also for the fast response you gave!!

**Comment by tboegi** [comment]
There is a new jar file  here:
https://github.com/tboegi/GpsMaster/releases/tag/v0.63.39-rc0
@IzzySoft : Could you test it, please (it works for me)
@rfuegen : thanks

**Comment by IzzySoft** [comment]
You're too fast :speak_no_evil: I wanted to wait patiently… :see_no_evil: :rofl:

Wonderful! Works fine, accepts the `*.gpx` passed at the command line – and loads it fine! \o/ I must say that from all viewers/editors I tried, GpsMaster has the best looking track display; with GPX Viewer I cannot even see waypoints (just the track – and not even that if only a single track point lacks time stamps), with Viking I have to manually load the map (as the default one always fails) and then have a hard time seeing the track… So I'm glad I found this!

So thanks a lot for solving this issue that fast! As it's solved, I'll close it now.
